assets/js/lib/test-utils/factories/sapSystems.js

```diff
@@ -36,6 +36,8 @@ export const sapSystemFactory = Factory.define(({ params }) => {
   const id = params.id || faker.string.uuid();
   const sid = params.sid || faker.string.alphanumeric(3, { casing: 'upper' });
   const databaseId = params.database_id || faker.string.uuid();
+  const databaseSid =
```

Comment:
maybe it is time at some point to have a sid generator function eheh
nah, forget about it in this PR at least. 
if you want, you can work afterwards on that as small tech-debt.
But it is not a big deal anyway